roles/tas_single_node/tasks/os.yml

```diff
@@ -1,4 +1,22 @@
 ---
+- name: Fail if the OS is not RHEL
+  ansible.builtin.fail:
+    msg: "Unsupported OS. This playbook supports only RHEL."
+  when: ansible_facts['os_family'] != 'RedHat'
+
+- name: Fail if the OS version is lower than 9.2
+  ansible.builtin.fail:
+    msg: "Unsupported OS version. This playbook supports only RHEL 9.2 or higher."
+  when: (ansible_facts['distribution'] == 'RedHat' and
```

Comment:
Will this let us run on CentOS as well? I think allowing CentOS is useful as well.